docs/userguide.md

```diff
@@ -128,76 +143,128 @@ Status:
 
 Follow these instructions for running an example to verify your deployment is
 working. The example will create a test namespace with a
-`federatednamespaceplacement` resource as well as federated template,
-override, and placement resources for the following k8s resources: `configmap`,
-`secret`, `deployment`, `service` and `serviceaccount`. It will then show how
-to update the `federatednamespaceplacement` resource to move resources.
+`federatednamespaceplacement` resource as well as federated template, override,
+and placement resources for the below k8s resources. It will then show how to
+update the `federatednamespaceplacement` resource to move resources. So let's
+set a variable with all the resource types for convenience:
+
+```bash
+RESOURCE_TYPES="deployment ingress job secret service"
+```
 
 ### Create the Test Namespace
 
 First create the `test-namespace` for the test resources:
 
 ```bash
-kubectl apply -f example/propagation/federatednamespace-template.yaml \
-    -f example/propagation/federatednamespace-placement.yaml
+kubectl apply -f example/propagation/federatednamespace-template.yaml
+```
+
+### Create the Test Namespace Placement
+
+Now we create the `test-namespace` `federatednamespaceplacement` resource.
+First replace the `null` string with the `HOST_CLUSTER` so that the federated
+controller does not prematurely delete the namespace, and all its contained
+resources, before we are able to update the list of `clusterNames`.
+
+```bash
+sed -i "s/null/${HOST_CLUSTER}/" example/propagation/federatednamespace-placement.yaml
+```
+
+Then create the `test-namespace` placement resource:
+
+```bash
+kubectl apply -f example/propagation/federatednamespace-placement.yaml
+```
+
+### Update FederatedNamespacePlacement
+
+Add both clusters via a patch command or manually:
+
+```bash
+kubectl -n test-namespace patch federatednamespaceplacement test-namespace \
+    --type=merge -p "{\"spec\":{\"clusterNames\": [\"${HOST_CLUSTER}\", \"${MEMBER_CLUSTER}\"]}}"
+
+kubectl -n test-namespace edit federatednamespaceplacement test-namespace
```

Comment:
Why edit again here? Do you mean `get` to check if the `federatednamespaceplacement` was updated successfully?

Reply:
This is just offering 2 ways to update the placement. One is via a patch command and the other is manually, as the comment right before it indicates.

Reply:
How about separate the two commands as follows:

Add both clusters via a patch command:

```bash
kubectl -n test-namespace patch federatednamespaceplacement test-namespace \
    --type=merge -p "{\"spec\":{\"clusterNames\": [\"${HOST_CLUSTER}\", \"${MEMBER_CLUSTER}\"]}}"
```

Or update manually:

```bash
kubectl -n test-namespace edit federatednamespaceplacement test-namespace
```